ProjectName/.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
If not in use, please delete it. On master, we always try to keep code in use only.